tests/targets/test_ios.py

```diff
@@ -196,13 +195,6 @@ def test_build_package_no_signature(self):
                 'You must fill the "ios.codesign.debug" token.'
             )
         ]
-        assert m_xcodebuild.call_args_list == [
```

Comment:
Because xcodebuild function signature has changed, instead of writing
```python
assert m_xcodebuild.call_args_list == [
    mock.call(
        "-configuration Debug",
        "-allowProvisioningUpdates",
        "ENABLE_BITCODE=NO",
        "CODE_SIGNING_ALLOWED=NO",
        None,
        "clean build",
        cwd="/ios/dir/myapp-ios",
    )
]
```
... I've moved the expectations to the m_cmd mock.